console/module/tag/src/main/java/org/eclipse/kapua/app/console/module/tag/server/GwtTagServiceImpl.java

```diff
@@ -198,9 +198,9 @@ public User call() throws Exception {
                 // Id", KapuaGwtCommonsModelConverter.convertKapuaId(tag.getScopeId())));
                 gwtTagDescription.add(new GwtGroupedNVPair("tagInfo", "tagName", tag.getName()));
                 gwtTagDescription.add(new GwtGroupedNVPair("entityInfo", "tagModifiedOn", tag.getModifiedOn().toString()));
-                gwtTagDescription.add(new GwtGroupedNVPair("entityInfo", "tagModifiedBy", modifiedUser.getName()));
+                gwtTagDescription.add(new GwtGroupedNVPair("entityInfo", "tagModifiedBy", modifiedUser != null ? modifiedUser.getName() : "N/A"));
```

Comment:
Please use `null`, this is managed by https://github.com/eclipse/kapua/blob/develop/console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/shared/model/GwtGroupedNVPair.java#L35 automatically